npm-frontend/src/TrackControls.jsx

```diff
@@ -112,29 +115,84 @@ class TrackControls extends Component {
 
   playMusic = () => {
     this.setState({ paused: false });
+    fetch('https://api.spotify.com/v1/me/player/play?device_id=beae69b8ce63eb244aa37e735390a02cb1d96d7c', {
```

Comment:
We probably want to do this without hardcoding if we have time to figure out another way

Reply:
Whoops, that device id doesn't actually have to be there. I meant to remove it. Thanks for catching it!